tests/unit/modin/test_unsupported.py

```diff
@@ -144,8 +144,6 @@ def test_unsupported_df(df_method, kwargs):
         ["prod", {}],
         ["ravel", {}],
         ["reorder_levels", {"order": ""}],
-        ["repeat", {"repeats": ""}],
```

Comment:
why those methods start becoming supported now? how are they supported now?

Reply:
if that is not a snowpark pandas claimed supported API, i thing we should overwrite in the extension to raise an unsupported error

Reply:
They are not newly supported, but they now fail at the query compiler layer rather than the frontend, and therefore need properties that are difficult to define on a mocked `SnowflakeQueryCompiler` object. They're now being checked in `tests/integ/modin/test_unimplemented.py`.